index.js

```diff
@@ -6,7 +6,7 @@ const arrify = require('arrify')
 const cachingTransform = require('caching-transform')
 const util = require('util')
 const findCacheDir = require('find-cache-dir')
-const fs = require('fs')
+const fs = require('fs-extra')
```

Comment:
Pulling in `fs-extra` just for `copySync` makes me sad.  That said I checked and using `cpy` increases the production install size by like 5MB (possibly due to using globby).  I'm not aware of a simple globless module that just does recursive copy.

I'm concerned about replacing the built-in `fs` with the `fs-extra` module.  I'd prefer that this instead:
```js
const fs = require('fs');
const { copySync } = require('fs-extra');
```

Reply:
It seemed like the best fit out of the modules that I looked at, not too big, good support and a lot of users.  It also had a copySync method that fits well with `nyc`.  `fs-extra.copySync` does have a big drawback in that it can't copy `./` to `./dest`, that the current `nyc instrument` implementation allows.  So this *might* disappear to be replaced with a custom solution.